### PR TITLE
Remove adblock.less

### LINKED ISSUE #9042: Remove adblock.less
## Test plan

https://github.com/brave/browser-laptop/pull/9043#issue-231371762

----

**Describe the issue you encountered:** Remove adblock.less.

- Extra QA steps:
    1.
    2.
    3.

- Screenshot if needed:

- Any related issues: #7320 #7321

### PR DESCRIPTION
**Pull request by luixxiul**
`customRightTextClassName`

Closes #9042
Addresses #7321 

Test Plan:
1. Open about:adblock
2. Open devtool
3. Make sure there is 15 px margin-left on the switches

Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

Test Plan:

Reviewer Checklist:

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

File: js/about/adblock.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ class AdBlockItem extends ImmutableComponent {
       <SwitchControl id={this.props.resource.get('uuid')}
         rightText={this.props.resource.get('title')}
         className={`switch-${this.props.resource.get('uuid')}`}
+        customRightText={css(styles.adblockLists__adblockItem__switchControl)}
```
Comment (bbondy) [inline js/about/adblock.js]: This name is a bit misleading for passing in styles, Maybe call it customRightTextStyle?

Comment (luixxiul) [inline js/about/adblock.js]: I would agree. I'm fixing other `custom*` with this PR later.

**Comment by luixxiul** [comment]
I confirmed pulling the two commits to `0.17` branch fix the conflicts for #9078.

https://github.com/brave/browser-laptop/pull/9043/files#diff-f9e9c72cdac7b145336979a3e76f20ebR195

**Comment by srirambv** [comment]
Not seeing the fix on 0.17.3
image

**Comment by luixxiul** [comment]
because the margin is this one:

screenshot 2017-06-21 12 05 56

**Pull request merged**